tanner/emulators/rfi.py

```diff
@@ -56,20 +56,20 @@ async def download_file(self, path):
                     rfile.write(data.encode('utf-8'))
         return file_name
 
-    def download_file_ftp(self, url):
+    async def download_file_ftp(self, url):
         host = url.host
         ftp_path = url.path.rsplit('/', 1)[0][1:]
         name = url.name
         try:
-            ftp = ftplib.FTP(host)
-            ftp.login()
-            ftp.cwd(ftp_path)
+            ftp = aioftp.Client()
+            await ftp.connect(host)
+            await ftp.login()
+            await ftp.change_directory(ftp_path)
```

Comment:
according to aioftp doc, `download` function can receive path, can we drop `change_directory` function?

Reply:
Then in the download function we have to pass url.path() as an argument instead of url.name()?